Dockerfile

```
@@ -5,7 +5,7 @@ FROM --platform=$BUILDPLATFORM golang:1.26.6-alpine AS builder
 ARG TARGETPLATFORM
 ARG BUILDPLATFORM
 
-RUN apk add --no-cache build-base git clang lld openssh-client bash
+RUN apk add --no-cache build-base git bash clang lld openssh-client
```

Hadolint (2.15.1)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)